BoothDownloader/src/Miscellaneous/Utils.cs

```diff
@@ -1,15 +1,14 @@
 using System.Collections.Concurrent;
 using ShellProgressBar;
+using BoothDownloader.Web;
 
 namespace BoothDownloader.Miscellaneous;
 
 public static class Utils
 {
     public static async Task DownloadFileAsync(string url, string destinationPath, IProgress<double> progress, CancellationToken cancellationToken = default)
     {
-        using var httpClient = new HttpClient();
-        httpClient.DefaultRequestHeaders.Add("User-Agent", BoothDownloader.UserAgent);
-        var response = await httpClient.GetAsync(url, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
+        var response = await BoothHttpClientManager.HttpClient.GetAsync(url, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
```

Comment:
Okay, now we were previously pretty mush using an anonymous httpclient, do we want to still do that or do we want to switch to an authed one (which is what's happening here)

Reply:
Wouldn't this be anonymous or paid depending if the user has authed?

Reply:
Yes, but previously it was always anonymous since we made a new HttpClient. Do we want to keep it that way? Or switch to authed?

Reply:
as long as the user agent is in then it doesn't matter to me. I just want to provide a courteous gesture to booth.

Reply:
If you want to generally keep the previous behavior, switch to `BoothHttpClientManager.AnonymousHttpClient`.
I don't know what suddenly introducing an auth cookie to paralleled downloads would introduce.

Reply:
alright ill do that